lib/ActionField.js

```diff
@@ -30,7 +30,7 @@ class ActionField extends Immutable.Record({
 			map.set('name', json.name || map.name);
 			map.set('title', json.title || map.title);
 			map.set('type', json.type || map.type);
-			map.set('value', json.value || map.value);
+			map.set('value', (json.value === false || json.value === 0) ? json.value : json.value || map.value);
```

Comment:
This is an interesting check.  This is basically *only* stopping `undefined` and empty strings thru.  Why do we need to restrict empty string?  And the sneaky part is that `null` is getting thru because of m.value?